scripts/arangodb-docker-container-setup/docker-container-scripts/test-data/testData.json

```diff
@@ -537,11 +555,50 @@
             ],
             "note": "this is a song"
         },
+        {
+            "_to": "media_items/1",
```

Comment:
Manually generate dual edge connections and update the generator utility to take an 'exceptions' list.

Reply:
Done.

Reply:
It's the dreaded 
```
Type 'InstanceFactory<Term>' is not assignable to type 'InstanceFactory<TResource>'. 
    Type 'Term' is not assignable to type 'TResource'. 
        'Term' is assignable to the constraint of type 'TResource', but 'TResource' could be instantiated with a different subtype of constraint 'Resource'.
```

Reply:
BTW Let's fight this battle another day.

Reply:
done